VideoLocker/res/values/strings.xml

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">edX</string>
```

Comment:
Oh, I guess I was having an issue because I was compiling on `dev` flavor then (which was missing the `resValue` setting.